tsconfig.server.json

```diff
@@ -3,7 +3,7 @@
         "outDir": "./dist",
         "target": "es6",
         "module": "commonjs",
-        "jsx": "react",
```

Comment:
To avoid annotation typescript error when import React from "react" is not written at top of file